lib/ts/unit-tests/test_MemArena.cc

```diff
@@ -92,6 +94,20 @@ TEST_CASE("MemArena freeze and thaw", "[libts][MemArena]")
 
 TEST_CASE("MemArena helper", "[libts][MemArena]")
 {
+  struct Thing {
+    int ten{10};
+    std::string name{"name"};
+
+    Thing() {}
+    Thing(int x) : ten(x) {}
+
+    Thing(std::string const &s) : name(s) {}
+
+    Thing(int x, std::string_view s) : ten(x), name(s) {}
+
+    Thing(std::string const &s, int x) : ten(x), name(s) {}
```

Comment:
Needed here, or add a test case below?

Reply:
Possibly add a test later, not sure what I did that.

Reply:
Same, if it's dead code let's not add it.

Reply:
After the latest rework, it still seems this constructor is not used for anything. Let's remove it if we don't need it.
It looks like the others actually are used in some test.

Reply:
More dead code after the latest branch update. I figure it's easier just to patch it: https://github.com/SolidWallOfCode/trafficserver/pull/24

Reply:
It's not dead, these are used in the test, although I got one of them backwards. It will be updated in the next commit.